src/ResourceManager/Resources/Commands.Resources/Microsoft.Azure.Commands.Resources.dll-Help.xml

```diff
@@ -4829,6 +4846,28 @@ Resources</dev:code>
           </command:commandLine>
         </command:commandLines>
       </command:example>
+      <command:example>
+        <maml:title>--------------------------  Example 2: Use a custom template uri and parameter uri with Incremental mode --------------------------</maml:title>
+        <maml:introduction>
+          <maml:paragraph>PS C:\&gt;</maml:paragraph>
+        </maml:introduction>
+        <dev:code>PS C:\&gt;New-AzureRmResourceGroupDeployment -Name testDeployment -ResourceGroupName ContosoEngineering -TemplateUri https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/101-vm-simple-windows/azuredeploy.json -TemplateParameterUri https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/101-vm-simple-windows/azuredeploy.parameters.json -Mode Incremental -DeploymentDebugSetting All</dev:code>
+        <dev:remarks>
+          <maml:para>This command creates a new deployment by using a custom template and a template file on from the quickstart gallery on github. The command uses the TemplateUri parameter to specify the template and the TemplateParameterUri to specify a file of parameters and parameter values.</maml:para>
```

Comment:
Instead of "file of Parameter" - Parameter file (which matches with our documentation on azure.com)